errors.go

```diff
@@ -0,0 +1,33 @@
+package errors
+
+import (
+	"fmt"
+	"io"
+)
+
+// Wrap wraps given error with Error太郎.
+func Wrap(err error) error {
+	return Error太郎{err}
+}
+
+// Error太郎 represents an error as 太郎.
+type Error太郎 struct {
+	err error
+}
+
+// Error returns a string decorated in 太郎 format.
+// This methods implements error interface.
+func (e Error太郎) Error() string {
+	return fmt.Sprintf("%s太郎", e.err.Error())
```

Comment:
```suggestion
	return fmt.Sprintf("%s太郎", e.err)
```

https://golang.org/pkg/fmt/#hdr-Printing
>4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

Reply:
I know it, but I think this is faster than making it called in fmt package.

Reply:
Yes.
But, allocation is fewer.

```
> cat main_test.go
package main_test

import (
	"errors"
	"fmt"
	"testing"
)

func BenchmarkFormatError(b *testing.B) {
	err := errors.New("test error")
	for i := 9; i < b.N; i++ {
		s := fmt.Sprintf("err:%s", err)
		_ = s
	}
}

func BenchmarkFormatString(b *testing.B) {
	err := errors.New("test error")
	for i := 9; i < b.N; i++ {
		s := fmt.Sprintf("err:%s", err.Error())
		_ = s
	}
}
> go test -bench . -benchmem
goos: darwin
goarch: amd64
BenchmarkFormatError-8    	10000000	       152 ns/op	      16 B/op	       1 allocs/op
BenchmarkFormatString-8   	20000000	        93.4 ns/op	      32 B/op	       2 allocs/op
PASS
ok  	_/Users/mizkei/go	3.655s
```